django_scheduled_tasks/models.py

```diff
@@ -35,6 +35,9 @@ def execute(self):
             func()
```

Comment:
Potentially calling `connection.close_if_unusable_or_obsolete()` here might fix the issue, but that has not been tested.